src/liberty/multiStepInput.ts

```diff
@@ -0,0 +1,366 @@
+import { QuickPickItem, window, Disposable, QuickInputButton, QuickInput, ExtensionContext, QuickInputButtons } from 'vscode';
+import * as devCommands from "./devCommands";
+import { getProjectOptions } from '../definitions/starterOptions';
+import * as vscode from "vscode";
+import * as fs from "fs";
+
+export async function multiStepInput(context: ExtensionContext) {
+
+	const projectOptions = await getProjectOptions();
+	const buildTools: QuickPickItem[] = projectOptions.b.options
+		.map(label => ({ label }));
```

Comment:
Typescript will complain that `Parameter 'label' implicitly has an 'any' type`. 
Should be `.map((label: any) => ({ label }));`

You can use the quick-fix in VS Code to fix these:
<img width="438" alt="image" src="https://user-images.githubusercontent.com/26146482/180849589-8446a669-d41c-43ba-a30a-0b02251a334d.png">